packages/server/src/models/EventModel.ts

```diff
@@ -33,4 +34,11 @@ export default class EventModel extends BaseModel<Event> {
 			.first();
 	}
 
+	public async deleteOldEvents(beforeMillis: number = Date.now() - Week) {
```

Comment:
All time should be milliseconds so there's no need to append a suffix. It's only in the rare case when it's not milliseconds that we append one.

No need for the weird default argument value, please make it required.